app/src/main/java/com/kks/nimblesurveyjetpackcompose/model/SurveyAnswer.kt

```diff
@@ -6,9 +6,8 @@ data class SurveyAnswer(
     val id: String,
     var text: String,
     val displayOrder: Int,
-    var selected: Boolean = false,
-    var isIdOnlyAnswer: Boolean = false
+    var selected: Boolean = false
 )
 
-fun SurveyAnswer.toSurveyAnswerRequest(): SurveyAnswerRequest =
+fun SurveyAnswer.toSurveyAnswerRequest(isIdOnlyAnswer: Boolean): SurveyAnswerRequest =
     if (isIdOnlyAnswer) SurveyAnswerRequest(id = id) else SurveyAnswerRequest(id = id, answer = text)
```

Comment:
Why not make `text` nullable instead? Then if `text` is null treat it as ID only since there is no text to put as answer 🤔 
```suggestion
fun SurveyAnswer.toSurveyAnswerRequest() = SurveyAnswerRequest(id = id, answer = text)
```

Reply:
The thing is it is not null. So even if we make it nullable, api response has text in it. So we need to exclude the text by ourselves.

Reply:
🤯 
What does the api return for that text? 🤔

Reply:
```
 {
            "id": "c5e8fc3bca388ec15884",
            "type": "answer",
            "attributes": {
                "text": "1",
                "help_text": null,
                "input_mask_placeholder": null,
                "short_text": "answer_1",
                "is_mandatory": false,
                "is_customer_first_name": false,
                "is_customer_last_name": false,
                "is_customer_title": false,
                "is_customer_email": false,
                "prompt_custom_answer": false,
                "weight": null,
                "display_order": 0,
                "display_type": "default",
                "input_mask": null,
                "date_constraint": null,
                "default_value": null,
                "response_class": "answer",
                "reference_identifier": null,
                "score": 0,
                "alerts": []
            }
        },
```
Something like this, `1` for the first and `2` for the second, and so on.

Reply:
Does the submission fail if we send that text on a smiley answer? 🤔

Reply:
Nope, it doesn't. But I think the `text` is just for some answers like questions with text fields.

Reply:
Just thinking we could simplify by quite a lot if we don't handle `idOnlyAnswer` case 🤔